src/supertux/menu/options_menu.cpp

```diff
@@ -179,6 +183,77 @@ OptionsMenu::OptionsMenu(bool complete) :
     resolutions.push_back(fullscreen_size_str);
   }
 
+  // Sound Volume
+  sound_volumes.clear();
```

Comment:
Make this allow an arbitrary float within a defined range. Limiting the user's options does not make any sense.